src/openrct2/Context.cpp

```diff
@@ -350,6 +350,24 @@ namespace OpenRCT2
                 config_save_default();
             }
 
+            try
+            {
+                _localisationService->OpenLanguage(gConfigGeneral.language);
+            }
+            catch (const std::exception& e)
+            {
+                log_error("Failed to open configured language: %s", e.what());
+                try
+                {
+                    _localisationService->OpenLanguage(LANGUAGE_ENGLISH_UK);
+                }
+                catch (const std::exception&)
```

Comment:
Missing exception name here, causes to relog original exception

Reply:
(note it was already doing that, but since you're moving it, we can take the chance to fix it)